Engine/Component/ComponentCollider.cpp

```diff
@@ -418,6 +434,76 @@ void ComponentCollider::SetVelocity(float3& velocity, float speed)
 
 }
 
+void ComponentCollider::SetVelocityEnemy(float3 & velocity, float speed)
+{
+	//bottom of the model
+	btVector3 bottom = body->getWorldTransform().getOrigin();
+	bottom.setY(bottom.getY() - 200 * box_size.getY());
```

Comment:
What does this number mean?

Reply:
It is used to make bigger the ray in the raycast of bullet to detect the ground. At 5 should be enough.